### ISSUE TITLE
README does not mention lapack dependency

### ISSUE THREAD

**Issue by eivindjahren** [equinor/ert#1962]
**Describe the bug**
When following the readme instructions, if lapack is not installed, you will get a reasonably complicated error message:

```
 Call Stack (most recent call first):
      /tmp/pip-build-env-hbxekqrt/overlay/lib/python3.6/site-packages/cmake/data/share/cmake-3.21/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
      /tmp/pip-build-env-hbxekqrt/overlay/lib/python3.6/site-packages/cmake/data/share/cmake-3.21/Modules/FindBLAS.cmake:1162 (find_package_handle_standard_args)
      /tmp/pip-build-env-hbxekqrt/overlay/lib/python3.6/site-packages/cmake/data/share/cmake-3.21/Modules/FindLAPACK.cmake:227 (find_package)
      /tmp/pip-build-env-hbxekqrt/overlay/lib/python3.6/site-packages/cmake/data/share/cmake-3.21/Modules/FindLAPACK.cmake:252 (_lapack_find_dependency)
      lib/CMakeLists.txt:228 (find_package)
```

The more informative part is hidden in `_skbuild/linux-x86_64-3.6/cmake-build/CMakeFiles/*.log` .
It would be useful to list os dependencies in the README.

**To Reproduce**
1. `pip install ert`
2. `ert gui my_config.ert`
3. Run experiment (IES/Smoother/ESMDA/Test)
4. …

**Expected behavior**

**Screenshots**

**Enviromment**
 - OS: [e.g. RHEL7]
 - ERT/Komodo Release: [e.g. 2.18 or 2020.12.00]
 - Python version
 - Remote/HPC execution involved: [yes|no]

**Additional context**

**Comment by sregales-TNO** [comment]
cmake is also not mentioned as a pre-requirement and also not present in dev-requirement.txt. Should we add cmake to the dev-requirement.txt and inform the user to first run 
```bash
pip install -r dev-requirement.txt
# and then
pip install -e .
``` 
or should we leave it off dev-requirements.txt and added together with lapack under the pre-requirements heading

**Comment by jondequinor** [comment]
> cmake is also not mentioned as a pre-requirement and also not present in dev-requirement.txt.

This is wrong, it's just that it is mentioned in https://github.com/equinor/ert/blob/main/pyproject.toml instead. Which is conforming to https://www.python.org/dev/peps/pep-0518/
